src/components/transactions/transactions.pug

```diff
@@ -32,18 +31,20 @@ md-card.offline-hide
                 span.tx(ng-switch-when='6') Send Lisk to Blockchain Application
                 span.tx(ng-switch-when='7') Send Lisk from Blockchain Application
                 span(ng-switch-default)
-                  span(ng-bind='transaction.senderId', ng-if='transaction.senderId !== $ctrl.account.get().address')
-                  span(ng-bind='transaction.recipientId', ng-if='transaction.senderId === $ctrl.account.get().address')
-              td(md-cell)
-                i.material-icons.expanding-button(ng-if='transaction.senderId !== $ctrl.account.get().address', data-show-send-modal, data-recipient-id='transaction.senderId') arrow_forward
-                i.material-icons.expanding-button(ng-if='transaction.senderId === $ctrl.account.get().address', data-show-send-modal, data-recipient-id='transaction.recipientId') arrow_forward
+                  span(ng-bind='transaction.senderId', ng-if='transaction.senderId !== $ctrl.account.get().address',
+                    data-show-send-modal, data-recipient-id='transaction.senderId', class='has-send-modal')
+                  span(ng-bind='transaction.recipientId', ng-if='transaction.senderId === $ctrl.account.get().address',
+                    data-show-send-modal, data-recipient-id='transaction.recipientId', class='has-send-modal')
+                  md-tooltip(md-direction='top', md-delay='350') Send to this recipient
               td(md-cell)
                 i.material-icons(ng-if='transaction.type === 0 && transaction.senderId === transaction.recipientId') replay
                 i.material-icons.in(ng-if='transaction.senderId !== $ctrl.account.get().address') call_received
                 i.material-icons.out(ng-if='transaction.type !== 0 || transaction.recipientId !== $ctrl.account.get().address') call_made
               td(md-cell)
-                .amount(ng-class='{ neutral:transaction.type === 0 &&  transaction.senderId === transaction.recipientId, positive: transaction.senderId !== $ctrl.account.get().address, negative: transaction.type !== 0 || transaction.recipientId !== $ctrl.account.get().address }')
+                .amount(ng-class='{"has-send-modal": transaction.amount > 0, neutral:transaction.type === 0 && transaction.senderId === transaction.recipientId, positive: transaction.senderId !== $ctrl.account.get().address, negative: transaction.type !== 0 || transaction.recipientId !== $ctrl.account.get().address }',
+                  data-show-send-modal, data-amount='transaction.amount')
```

Comment:
What I had in mind was to send the same amount not repeating the transaction. But I'm open to change it if you think that's gonna be more useful.